### PR TITLE
feat: adapt for probot v11. Use `@actions/core` for logging

### PR DESCRIPTION
**Pull request by gr2m**
closes https://github.com/probot/adapter-github-actions/pull/16

BREAKING CHANGE: package name has been renamed to "@probot/adapter-github-actions"

BREAKING CHANGE: the package now has the same export as `"probot"`.

Before

```js
const runProbot = require('probot-actions-adapter');
const app = require('./app');
runProbot(app);
```

After

```js
const { run } = require("@probot/adapter-github-actions");
const app = require("./app");
run(app);
```

-----
[View rendered LICENSE.md](https://github.com/probot/adapter-github-actions/blob/probot-v11/LICENSE.md)
[View rendered README.md](https://github.com/probot/adapter-github-actions/blob/probot-v11/README.md)

### CODE REVIEW

**Comment by gr2m** [comment]
@swinton this is basically a total rewrite, I'd appreciate if you could have a look. If we decide to merge, I'd deprecate the current `probot-actions-adapter` package on npm

File: README.md [hunk 1]
```diff
@@ -1,80 +1,66 @@
...
-// This also acts as the main entrypoint for the Action
-runProbot(app);
+run(app).catch((error) => {
+  console.error(error);
+  process.exit(1);
+});
 ```
 
-### Example `action.yml`
+Then use `index.js` as your entrypoint in the `action.yml` file
```
Comment (swinton) [inline README.md:34]: There's a bit more nuance here to unpack.

Users can either:

1. Commit their node_modules
2. Compile their JS down to a single executable script (e.g. via `@vercel/ncc` -- this is my preference)

Perhaps we can lean on the docs to unpack this nuance: https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#commit-tag-and-push-your-action-to-github

```suggestion
Follow the official  GitHub documentation for next steps including:

1. [Creating an action metadata file](https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#creating-an-action-metadata-file)
1. [Commit, tag, and push your action to GitHub](https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#commit-tag-and-push-your-action-to-github)
```

**Review by swinton — APPROVED** [review]
Thanks for all the work here, @gr2m. I love the new name as well! I had one README suggestion...

File: README.md (hunk 1 above)
Comment (gr2m) [reply README.md:34]: okay like this? https://github.com/probot/adapter-github-actions/pull/19/commits/6760bd2217eefc6060733ace2ac068a6e9adfd9b

**Review by jetersen — APPROVED** [review]
LGTM 👏

**Pull request merged**